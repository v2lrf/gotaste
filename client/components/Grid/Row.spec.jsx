import React from 'react'
import { Row } from './index'

describe('Row', () => {
  let component
  beforeEach(() => {
    component = render(<Row>See, I am inside a row!</Row>)
  })

  describe('Snapshots', () => {
    it('should match the snapshot', () => {
      expect(component).toMatchSnapshot()
    })
  })
})
