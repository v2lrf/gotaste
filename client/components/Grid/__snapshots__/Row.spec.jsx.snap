// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Row Snapshots should match the snapshot 1`] = `
<div
  class="px-2"
>
  <div
    class="flex -mx-2 flex-wrap"
  >
    See, I am inside a row!
  </div>
</div>
`;
